spot-client/src/common/backend/SpotBackendService.test.js

```diff
@@ -155,6 +167,20 @@ describe('SpotBackendService', () => {
                             .toBe(true);
                     });
             });
+            it('notifies the listeners of JWT update - when starting with the registration expired', () => {
+                dateSpy = jest.spyOn(Date, 'now');
+                dateSpy.mockReturnValue(Date.now() + MOCK_RESPONSE.expiresIn);
```

Comment:
Oh man what is this? I've been using lolex. This looks better.

Reply:
funny I was trying to find lolex usage, but was not able to find it :P